### PR TITLE
Update Lisk Commander reference - Closes #771

### LINKED ISSUE #771: Update the Lisk Commander documentation to v5
Update the Lisk Commander documentation, to reflect the latest changes in the Lisk SDK v5.

Branch: `docs-sdk`

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #771

### CODE REVIEW

File: modules/ROOT/pages/references/lisk-commander/index.adoc [hunk 1]
```diff
@@ -80,92 +77,14 @@ $ lisk # displays general help and the command list.
...
-  message               Commands relating to user messages.
-  network-identifier    Commands relating to the network identifier.
-  node                  Commands relating to Lisk node.
-  passphrase            Commands relating to Lisk passphrases.
-  transaction           Commands relating to Lisk transactions.
-  warranty              Displays warranty notice.
+  account     Commands relating to Lisk accounts.
+  help        Displays help.
+  message     Commands relating to user messages.
+  passphrase  Commands relating to Lisk passphrases.
```
Comment (ManuGowda) [inline modules/ROOT/pages/references/lisk-commander/index.adoc]: i see the hash-onion and network-identifier command missing?
```
 account             Commands relating to Lisk accounts.
  hash-onion
  help                Displays help.
  message             Commands relating to user messages.
  network-identifier
  passphrase          Commands relating to Lisk passphrases.
```

Comment (Tschakki) [reply modules/ROOT/pages/references/lisk-commander/index.adoc]: Good catch, added them now!

File: modules/ROOT/pages/references/lisk-commander/index.adoc [hunk 2]
```diff
@@ -80,92 +77,16 @@ $ lisk # displays general help and the command list.
...
-  delegate              Commands relating to Lisk delegates.
-  help                  Displays help.
-  message               Commands relating to user messages.
-  network-identifier    Commands relating to the network identifier.
-  node                  Commands relating to Lisk node.
-  passphrase            Commands relating to Lisk passphrases.
-  transaction           Commands relating to Lisk transactions.
-  warranty              Displays warranty notice.
+  account             Commands relating to Lisk accounts.
+  hash-onion
```
Comment (ManuGowda) [inline modules/ROOT/pages/references/lisk-commander/index.adoc]:
```suggestion
  hash-onion   Create hash onions to be used by the forger.
```

File: modules/ROOT/pages/references/lisk-commander/index.adoc [hunk 3]
```diff
@@ -80,92 +77,16 @@ $ lisk # displays general help and the command list.
...
+  help                Displays help.
+  message             Commands relating to user messages.
+  network-identifier
```
Comment (ManuGowda) [inline modules/ROOT/pages/references/lisk-commander/index.adoc]:
```suggestion
  network-identifier   Creates Network identifier for the given genesis block id and community identifier.
```

**Review by ManuGowda — DISMISSED** [review]

**Pull request merged**